providers/os/resources/discovery/container_registry/registry_test.go

```diff
@@ -49,7 +49,7 @@ func TestHarbor(t *testing.T) {
 		require.NoError(t, err, url)
 
 		// check that we resolved it correctly and we got a specific shasum
-		assert.Equal(t, "container-registry", a.Connections[0].Type)
+		assert.Equal(t, "registry-image", a.Connections[0].Type)
```

Comment:
i do not know why parsing an image was returning back `container-registry`, this looked wrong